src/classes/RD2_OpportunityEvaluationService.cls

```diff
@@ -213,6 +213,37 @@ public inherited sharing class RD2_OpportunityEvaluationService {
         } set;
     }
 
+    /**
+    * @description Contains all mandatory Recurring Donation fields required for the Opportunity Evaluation Service
+    */
+    private Set<String> rdFields = new Set<String> {
+        'Name',
```

Comment:
It might be worth adding other system fields into this list:

- Id
- CreatedDate
- CreatedById
- LastModifiedDate
- LastModifiedById